indico/modules/events/tracks/controllers.py

```diff
@@ -98,9 +117,15 @@ class RHSortTracks(RHManageTracksBase):
     def _process(self):
         sort_order = request.json['sort_order']
         tracks = {t.id: t for t in self.event.tracks}
-        for position, track_id in enumerate(sort_order, 1):
-            if track_id in tracks:
-                tracks[track_id].position = position
+        track_groups = {tg.id: tg for tg in self.event.track_groups}
+        for position, item in enumerate(sort_order, 1):
+            if item['type'] == 'track':
+                tracks[item['id']].position = position
+                parent = item['parent'] if item['parent'] != 'None' else None
```

Comment:
What about sending the data as `null` instead of `'None'` for this? Then you get a literal `None` here and can simplify this to `parent = item['parent']` (or even use `item['parent']` directly)